### PR TITLE
[LB, Spectrum] Update docs on using Spectrum with a load balancer

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
Triggered by PCX-6474

### CODE REVIEW

File: assets/images/spectrum/spectrum-tcp-check.png
Comment (RebeccaTamachiro) [inline assets/images/spectrum/spectrum-tcp-check.png:1]: Replaced this image as it contained a field that is no longer displayed. Did not add a new image but instead only mentioned the fields and how to fill in, in a similar format that we have elsewhere in Dev Docs

File: content/load-balancing/additional-options/spectrum.md [hunk 1]
```diff
@@ -8,6 +8,61 @@ meta:
...
+2. Select **Create Load Balancer**.
+
+3.  On the **Hostname** page, define the settings presented and select **Next**.
+    *   Enter a **Hostname**, which is the DNS name at which the load balancer is available. For more details on record priority, refer to [DNS records for load balancing](/load-balancing/load-balancers/dns-records/).
+    {{<Aside type="warning">}}
+  To prevent issues with DNS resolution, the load balancer hostname should be different from the hostname (or domain) you intend to define for your Spectrum application.
+    {{</Aside>}}
+    *   Keep the orange cloud icon enabled, meaning the load balancer is proxied. This refers to the [proxy mode](/load-balancing/understand-basics/proxy-modes/) and, with Spectrum, traffic is always proxied.
+    *   Keep **Session Affinity** disabled as this feature is not supported with Spectrum.
+
```
Comment (angelampcosta) [inline content/load-balancing/additional-options/spectrum.md:36]: There is also **Adaptive Routing**. You don't mention this one.

File: content/load-balancing/additional-options/spectrum.md [hunk 2]
```diff
@@ -8,6 +8,61 @@ meta:
...
+    *   Review the monitors attached to your pools.
+    *   If needed, you can attach an existing monitor or [create a new monitor](/load-balancing/monitors/create-monitor/#create-a-monitor).
+
+6.  On the **Traffic Steering** page, choose an option for [Traffic steering](/load-balancing/understand-basics/traffic-steering/steering-policies/) and select **Next**.
+
+7. Keep **Custom Rules** page empty as this feature is not supported with Spectrum.
+
+8. On the **Review** page:
+    *   Review your configuration and make any changes.
+        * If you set traffic steering to Off, re-order the pools in your load balancer to adjust the fallback order.
```
Comment (angelampcosta) [inline content/load-balancing/additional-options/spectrum.md]:
```suggestion
        * If you set traffic steering to **Off**, re-order the pools in your load balancer to adjust the fallback order.
```

File: content/spectrum/_partials/_spectrum-with-load-balancer-dash.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+_build:
+  publishResources: false
+  render: never
+  list: never
+---
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login).
+2. Select **Spectrum**.
+3. Select **Create an Application**. If this is your first time using Spectrum, the **Create an Application** modal appears.
+4. Select your **Application Type**.
```
Comment (angelampcosta) [inline content/spectrum/_partials/_spectrum-with-load-balancer-dash.md]:
```suggestion
4. Select your **[Application Type](/spectrum/reference/configuration-options/#application-type)**.
```

File: content/spectrum/about/load-balancer.md [hunk 4]
```diff
@@ -6,25 +6,32 @@ weight: 0
 
 # Cloudflare Load Balancer
 
-You can configure Spectrum and Cloudflare's Load Balancing to provide TCP healthchecks, failover, and traffic steering to bring resiliency to your Spectrum applications. To prevent issues with DNS resolution for a Spectrum application, do not use the same Spectrum hostname as a current Load Balancing hostname.
+You can configure Spectrum with Cloudflare [Load Balancing](/load-balancing/) to provide TCP healthchecks, failover, and traffic steering, bringing resiliency to your Spectrum applications.
```
Comment (angelampcosta) [inline content/spectrum/about/load-balancer.md:9]: Should it be Load Balancer, like the section title?

File: content/spectrum/about/load-balancer.md [hunk 5]
```diff
@@ -6,25 +6,32 @@ weight: 0
...
 
-{{<Aside type="note" header="Note">}}
-
-This feature requires an Enterprise plan. If you would like to upgrade, contact your account team.
-
-{{</Aside>}}
+For an overview of how Cloudflare Load Balancing works refer to [Load Balancing components](/load-balancing/understand-basics/load-balancing-components/). For setup guidance refer to [Add load balancing to Spectrum applications](/load-balancing/additional-options/spectrum/).
```
Comment (angelampcosta) [inline content/spectrum/about/load-balancer.md:11]: Same here.

**Review by angelampcosta — APPROVED** [review]

File: content/spectrum/about/load-balancer.md (hunk 4 above)
Comment (RebeccaTamachiro) [reply content/spectrum/about/load-balancer.md:9]: Not currently super consistent across the board, but basically the product is called `Load Balancing` and then there are `load balancers` which are a part/component of the Load Balancing solution. Pushing a commit soon to change the section title and h1

File: content/load-balancing/additional-options/spectrum.md (hunk 1 above)
Comment (RebeccaTamachiro) [reply content/load-balancing/additional-options/spectrum.md:36]: Well noted! Actually, this made me realize it has not been added to the generic steps about LB either. Will create a follow-up ticket to look into this. As for compatibility between this feature and Spectrum, found some info and updated accordingly - will get PM sign-off as well just to be extra sure.

**Pull request merged**